docs/eventing/README.md

```diff
@@ -73,25 +87,6 @@ subscription, and if you have any functions that make use of any external (to
 cluster) services, you must enable it also for them to work.
 [Follow the instructions to configure outbound network access](../serving/outbound-network-access.md).
 
-Install the core Knative Eventing (which provides an in-memory
```

Comment:
Why are these instructions removed? Are they now in-lined in "Fllow the instructions to install on the platform of your choice"?

Reply:
Yep, the install guides cover install steps for all the components (Eventing included). There's no reason to repeat the content on this page.